ConfirmPwd/ConfirmPwd.jsx

```diff
@@ -0,0 +1,64 @@
+import React from "react";
+import {
+  faCheck,
+  faTimes,
+  faInfoCircle,
+} from "@fortawesome/free-solid-svg-icons";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+
+export default function UserName({ userPwd, setUserPwd }) {
```

Comment:
can ConfirmPwd and Password reuse the same underlying PasswordText component? Seems redundant to me.